src/select/src/components/__tests__/__snapshots__/Creatable.test.js.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Creatable render matches snapshot 1`] = `
```

Comment:
`Creatable.test.js.snap` is identical to `Select.test.js.snap`, except for line 3. Is this intentional?

Reply:
Select and Creatable are separate components from the react-select library so I also kept their tests separate in case one of them is updated without the other.

Reply:
But why are the snapshots basically identical?